### DISCUSSION TITLE
How to prevent acid dosing when shocking the pool?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#991] [REM] [answered]
I'm shocking my pool before I head out for a month.

When I add a lot of chlorine, the ph reading is grossly inaccurate.  So, it wants to dose acid, even when I use the maximum ph setpoint of 8.0.

I figure I can prevent acid dosing by setting the tank level to 0.  Is there a better way?

Thanks!

**Comment by rstrouse** [comment, marked as answer]
Yes add a long mix time to allow the chlorine to cycle into its component parts.   I do this whenever I shock the pool.

**Reply by rstrouse** [reply]
To set a manual mix time click on the REM Chem line in the chemistry bubble.  There you will find a Mix Acid button.  This will set a mix for a specified period of time then resume its normal schedule.

**Comment by davthomaspilot** [comment]
Thanks!
